cryptoinvestor/views/assets.py

```diff
@@ -42,10 +42,33 @@ def get_objects(self):
                 }
             }
 
+        toasts = []
+        for asset in assets:
+            id_ = asset.get('id', 'N/A')
+            current_rate = asset.get('rate', 0.00)
+            try:
+                previous_rate = data.get(id_, {}).get(base, [])[-2].get('rate', 0.00)
+                value = round((current_rate - previous_rate), 4)
+            except IndexError:
+                value = 0
+
+            if value > 0:
+                message = f"From last time {id_} went up by {value}!"
+                color = 'green'
+            elif value < 0:
+                message = f"From last time {id_} went down by {value}!"
+                color = 'red'
+            else:
+                message = f"From last time {value} did not change!"
+                color = 'blue'
+
+            toasts.append(self.app.add_toast(message, color))
```

Comment:
This is good but I meant for you to export this whole block (from 46 to 65 lines)

Reply:
Should we make two different add_toast functions ? One for general toasts with simple message and another for more complex ones such as this one, when we want to differentiate the colors and texts ?

Reply:
It is good to have general add_toast method in app. But yes there should be another asset-reporting specific, at least a helper function, not necessarily assigned to a class.